lightkurve/periodogram.py

```diff
@@ -603,6 +606,10 @@ def from_lightcurve(lc, min_frequency=None, max_frequency=None,
         freq_unit : `astropy.units.core.CompositeUnit`
             Default: 1/u.day. The desired frequency units for the Lomb Scargle
             periodogram. This implies that 1/freq_unit is the units for period.
+        normalization : str
```

Comment:
I suggest making the two valid options explicit in the docstring:
```suggestion
        normalization : 'psd' or 'amplitude'
```